src/client/game.cpp

```diff
@@ -2257,7 +2257,9 @@ void Game::toggleDebug()
 	else if (state == 2)
 		m_game_ui->showTranslatedStatusText("Profiler graph shown");
 	else if (state == 3)
-		m_game_ui->showTranslatedStatusText("Wireframe shown");
+		m_game_ui->showTranslatedStatusText(driver->getDriverType() == video::EDT_OGLES2 ?
+				"Wireframe shown, but not supported in OpenGL ES" :
```

Comment:
at least I'd change it to "Wireframe not supported by video driver"
also: I believe `showTranslatedStatusText` is picked up by the gettext script so that ternary won't work

Reply:
https://github.com/minetest/minetest/blob/5bcb7983ec1648b9993e4092be014308726af383/util/updatepo.sh#L63-L65

Reply:
Done.